src/pages/Home/HomePageInternal/OwnedTokenGrid.tsx

```diff
@@ -38,13 +41,13 @@ export default function OwnedTokenGrid({
                     </h3>
                     <span
                       className={ClassNamesLogic(
-                        community?.communityData?.isActive ?? false
+                        community?.categories ?? false
                           ? 'text-green-800 bg-green-100'
                           : 'bg-pink-100 text-pink-800',
                         'flex-shrink-0 inline-block px-2 py-0.5' +
                           'text-xs font-medium  rounded-full'
                       )}>
-                      {community?.communityData?.isActive ? 'Active' : 'New'}
```

Comment:
I explicitly added the isActive because a community could be active even if it has no categories? Rellying on the category size to determine whether the community is active is strange. Honestly, to simplify our work, we should get rid of these default communities and only have custom ones.

Reply:
it was a very unnecessary added complexity, if the community doesn't exist then it is undefined anyway, a post can be created without a topic, categories will then be an empty array.

Reply:
maybe but we need to decide and plan it is a bit hard to develop without a clear goal